test/tasks/maintenance_tasks/task_test.rb

```diff
@@ -67,5 +68,20 @@ class TaskTest < ActiveSupport::TestCase
       message = "MaintenanceTasks::Task must implement `process`."
       assert_equal message, error.message
     end
+
+    test ".throttle_conditions inherits conditions from superclass" do
```

Comment:
This test might be unnecessary now that we're using `class_attribute` - let me know if we should 🔥  it